Sources/AppStoreConnectCLI/Services/Operations/InviteTesterOperation.swift

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct InviteTesterOperation: APIOperation {
+
+    struct InviteBetaTesterDependencies {
+        let apps: (APIEndpoint<AppsResponse>) -> Future<AppsResponse, Error>
```

Comment:
Should this not be
```suggestion
        let appsResponse: (APIEndpoint<AppsResponse>) -> Future<AppsResponse, Error>
```
for consistency?